easybuild/easyconfigs/x/xarray/xarray-2025.7.0-gfbf-2025a.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'PythonPackage'
+
+name = 'xarray'
+version = '2025.7.0'
+
+homepage = 'https://github.com/pydata/xarray'
+description = """xarray (formerly xray) is an open source project and Python package that aims to bring
+ the labeled data power of pandas to the physical sciences, by providing N-dimensional variants of the
+ core pandas data structures."""
+
+toolchain = {'name': 'gfbf', 'version': '2025a'}
+sources = [SOURCE_TAR_GZ]
+checksums = ['fd83ac8d638e7caef9d7f0c82bcdf380cede29d2ff84575133b2b95164af78ee']
+
+dependencies = [
+    ('Python', '3.13.1'),
+    ('SciPy-bundle', '2025.06'),
+]
+
+# fix license style in pyproject.toml
+preinstallopts = """sed -i 's/"Apache-2.0"/{text = "Apache-2.0"}/' pyproject.toml &&"""
+
```

Comment:
i think we want to go with #23302 instead of sed'ing or patching all the toml files